src/sentences/ttm.rs

```diff
@@ -118,6 +118,7 @@ pub struct TtmData {
     /// Set to true if target is a reference used to determine own-ship position or velocity
     pub is_target_reference: bool,
     /// Time of data
+    #[cfg_attr(feature = "defmt-03", defmt(Debug2Format))]
```

Comment:
is this diff intended?

Reply:
Yes, it is.
We were not testing defmt-03 and NaiveTime needs the Debug2Format as chrono doesn't have defmt impls.

Sadly, Windows build fails so I had to tweak the CI workflow.